### PR TITLE
Add a ProtonVPN Road Warrior setup page

### LINKED ISSUE #516: Add WireGuard Road Warrior Setup for ProtonVPN
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [ *] I have read the contributing guide lines at https://github.com/opnsense/docs/blob/master/CONTRIBUTING.md
- [ *] I have searched the existing issues and I am convinced that mine is new.

**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you like**

Adding clear and concise documentation in the style of https://docs.opnsense.org/manual/how-tos/wireguard-client-mullvad.html, https://docs.opnsense.org/manual/how-tos/wireguard-client-azire.html but for ProtonVPN

**Describe alternatives you considered**
N/A

**Additional context**
ProtonVPN is a popular choice for a VPN in the Open Source community so it would be nice if this existed.

### PR DESCRIPTION
**Pull request by satrapes**
Add images from ProtonVPN website
Add instructions on how to setup the ProtonVPN with WireGuard
Fix typos in wireguard-selective-routing.rst
Add wireguard-client-proton.rst and wireguard-selective-routing.rst to toctree to remove warnings
Closes #516

### CODE REVIEW

File: source/manual/how-tos/wireguard-client-proton.rst [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+.. Note::
+    The UI for configuring the Instances and Peers changed with OPNsense verion 23.7.9 so some of the fields may be in different
+    places.
+
+--------------------------
+Step 4 - Turn on WireGuard
+--------------------------
+
+Turn on WireGuard under :menuselection:`VPN --> WireGuard --> General` if it is not already on
```
Comment (AdSchellevis) [inline source/manual/how-tos/wireguard-client-proton.rst:177]: VPN --> WireGuard --> Settings --> General

**Review by AdSchellevis — CHANGES_REQUESTED** [review]
some small remarks, but other than that this looks good. @satrapes thanks!

File: source/manual/how-tos/wireguard-client-proton.rst [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+-----------------------------------------
+
+The configuration is available in the ProtonVPN website.
+The `landing page <https://account.protonvpn.com/dashboard>`__ appears after signing in. 
+Click on Downloads from the left hand panel or go to the `downloads page <https://account.protonvpn.com/downloads>`__ and
+scroll down to the `Wireguard configuration <https://account.protonvpn.com/downloads#wireguard-configuration>`__
+
+The existing Wireguard configurations appear first with their expiration dates and following are the options to generate new ones.
+
+.. image:: images/proton_wireguard_configuration.png
```
[the remark this answers is not in the document]
Comment (satrapes) [reply source/manual/how-tos/wireguard-client-proton.rst:24]: You have a point there. I will only leave a couple. The initial page and the one that shows the downloaded configuration.

File: source/manual/how-tos/wireguard-client-proton.rst (hunk 1 above)
Comment (satrapes) [reply source/manual/how-tos/wireguard-client-proton.rst:157]: Fixed

File: source/manual/how-tos/wireguard-client-proton.rst [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+.. code-block:: sh
+
+    wg pubkey < wgPrivateKey > wgPublicKey.pub
+
+----------------------------------
+Step 3 - Setup WireGuard Instance
+----------------------------------
+
+- Go to :menuselection:`VPN --> WireGuard --> Instances`
```
[the remark this answers is not in the document]
Comment (satrapes) [reply source/manual/how-tos/wireguard-client-proton.rst]: Fixed

**Comment by satrapes** [comment]
I think i'm all done

**Comment by AdSchellevis** [comment]
@satrapes thanks, I'll merge it and double check when compiling later.

**Pull request merged**